python-packages/presto_viewmodels_drf/presto_viewmodels_drf/serializers.py

```diff
@@ -0,0 +1,46 @@
+class OptinFieldsSerializerMixin:
+    """
+    Regulates fields exposed by default & as requested.
+
+    Construct URLs like /?include_fields=first_name,email to opt-in into fields.
+
+    By default, all "opt_in_only_fields" are excluded from serializer, unless specifically opted-in.
+
+    If any of fields specified to be included does not belong to opt_in_only_fields, the serializer
+    will perform in "strict" mode that ONLY fields specced are included.
```

Comment:
This sentence is confusing

Isn't it just that if you specify any fields then only those fields are included (ie. it's irrelevant what it's opt_in_only_fields - that's only used if you don't specify  anything)?

Reply:
Perhaps just

> If `include_fields` is specified then only those fields are serialized.